server/build.gradle.kts

```diff
@@ -34,6 +35,7 @@ dependencies {
     testImplementation("io.vertx:vertx-web-client")
     testImplementation("org.mockito:mockito-core")
     testImplementation("org.mockito:mockito-junit-jupiter")
+    testImplementation("org.mockito:mockito-inline:2.13.0")
```

Comment:
What's this used for? Internet tells me "mockito-inline for some experimental features like mocking final classes and methods, mocking static methods etc" which sounds like a bad testing practice

Reply:
trace was a bit hard to read but I believe mockito could not mock mapToTimerFunction in metadataTimerDatabase

Reply:
I don't see test trying to read it. Is this needed anymore? It's a private function so I don't see we shouldn't rely on it in tests.

Reply:
today I learned kotlin really like final classes, mockito not so much